src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+}
+
+.stars-star {
+
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-size: center;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
Make sure that the last elements of the row don't have margin-right
![image](https://github.com/mate-academy/layout_stars/assets/96409563/ce373997-085e-4187-aac9-a414643c95ca)